cg/meta/workflow/mutant/quality_controller/utils.py

```diff
@@ -0,0 +1,106 @@
+from cg.apps.lims.api import LimsAPI
+from cg.constants.constants import MutantQC
+from cg.constants.lims import LimsArtifactTypes, LimsProcess
+from cg.exc import CgError, LimsDataError
+from cg.meta.workflow.mutant.metrics_parser.metrics_parser import MetricsParser
+from cg.meta.workflow.mutant.metrics_parser.models import SampleResults
+from cg.meta.workflow.mutant.quality_controller.models import (
+    MutantPoolSamples,
+    QualityMetrics,
+)
+from genologics.entities import Artifact
+from genologics.entities import Sample as LimsSample
+from cg.models.cg_config import LOG
```

Comment:
Some imports are not used here. We have some [hooks](https://atlas.scilifelab.se/infrastructure/tech_stack/editors/pycharm/#install-pre-commit) that will fix it for you. And pycharm also have some commands you could use: https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html#reformat_file
```suggestion
```